otter/test/models/test_cass_models.py

```diff
@@ -411,10 +413,23 @@ def _responses(*args):
         self.policies = []
         self.mock_log = mock.MagicMock()
 
-        self.kz_client = mock.Mock()
-        self.lock = self.mock_lock()
-        self.kz_client.Lock.return_value = self.lock
-        self.kz_client.delete.return_value = defer.succeed('something else')
+        self.kz_client = ZKCrudModel()
```

Comment:
I wish there were some kind of fake verification here; a check for the consistency between `kz_client` here and what it is in real life.  It looks right to me I just wish I didn't have to manually check :)